frontend/src/components/floating-menus/MenuList.vue

```diff
@@ -6,44 +6,51 @@
 		:type="'Dropdown'"
 		:windowEdgeMargin="0"
 		:escapeCloses="false"
-		v-bind="{ direction, scrollableY, minWidth }"
+		v-bind="{ direction, minWidth, scrollableY: !fixedHeight && scrollableY }"
```

Comment:
Nitpick, feel free to ignore since it's so minor: it might be slightly more readable and consistent to put the `scrollableY` before the `&&` (i.e. swap the two sides of the `&&`).